src/Turbo/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.23.0
+
+-   Add support for providing options to the EventSource via `turbo_stream_listen`
```

Comment:
I think we should only use the term "authenticated" or "credential based" event source... because the options is not open to all transports nor all keys